### ISSUE TITLE
Add in dependancies laravel/promt

### ISSUE THREAD

**Issue by agoalofalife** [aarondfrancis/solo#25]
Hello, I decided to try your new package and downloaded for laraval `10.48.4`
I got the error message 
`Undefined constant Laravel\Prompts\Key::SHIFT_DOWN`

SHIFT_DOWN was added in promts in [v0.1.21](https://github.com/laravel/prompts/releases/tag/v0.1.21)
I think it's better to add dependency in composer.json that you use promts starts with this version

In laravel `10.48.4`
composer.json has 

```
   "laravel/prompts": "^0.1.9",
```

**Comment by simonhamp** [comment]
**tl;dr: From experience, I'm finding that getting the right version constraint for `laravel/prompts` is tricky while Prompts is still pre-v1.**

It would _currently_ need a constraint like `^0.1.21|^0.2|^0.3`, but this would need to be kept on top of.

Why? Folks may also add `laravel/prompts` to their own app as a direct dependency when their app depends on it (as they should!) - at a version that makes sense for them.

So when Prompts gets a new release, and the good developer tries to upgrade their dependency, if the maintainers of their _other_ dependencies-that-rely-on-Prompts haven't updated _their_ constraints, the developer will get a version conflict.

That's the behaviour we want from Composer, but it can happen quite a bit while a package is in pre-release.

With Prompts it's happening a bit more often _at the moment_ because [they're not strictly following Composer's interpretation of SemVer](https://github.com/laravel/framework/discussions/53158) (this is not a complaint, just a statement of fact 😊).

In practice, I'm finding this needs to be done relatively soon after any new release of Prompts, as folks are quite keenly using it despite it being pre-release (I don't blame them, it's great!)

But as a maintainer, I think this is a bit of a pain.

It can be side-stepped by using constraints like `>=0.1.21` or `*`, but as a good citizen/dependency, if you care about not breaking your user's apps, then these are way too permissive.

Caveat all of this with the real possibility that the release cycle for Prompts is likely to improve going forwards.

---

### Aside: Composer pre-release version constraints

When using a constraint that starts with `^0.`, [Composer will treat this as pre-release](https://getcomposer.org/doc/articles/versions.md#caret-version-range-) and shift its concept of the `major` version number down to the second digit.

**Comment by aarondfrancis** [comment]
Thank you both! I think this is more correct now.
